homeassistant/components/sms/strings.json

```
@@ -0,0 +1,15 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Connect to the modem",
+        "data": { "device": "Device" }
+      }
+    },
+    "error": {
+      "cannot_connect": "Failed to connect to modem, please try again",
+      "unknown": "Unexpected error"
+    },
+    "abort": { "already_configured": "The modem is already configured" }
```

We need to add an abort string for `"single_instance_allowed"`.

Please use as many common strings as possible. Here are the common strings available:
https://github.com/home-assistant/core/blob/3ae4eba457160b22bf53fbc6383705c6947efefe/homeassistant/strings.json#L19-L57

```suggestion
    "abort": {
      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]"
    }
    
```